libraries/standard/mqtt/src/mqtt.c

```diff
@@ -665,28 +729,17 @@ static MQTTStatus_t handleIncomingAck( MQTTContext_t * const pContext,
                 status = sendPublishAcks( pContext,
                                           packetIdentifier,
                                           &publishRecordState );
-
-                if( status == MQTTSuccess )
-                {
```

Comment:
Was invocation of `pContext->callbacks.appCallback` removed for the buffer corruption issue from `sendPublishAcks`?